plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/translators/TranslatorContext.cpp

```diff
@@ -419,8 +425,27 @@ void TranslatorContext::deserialise(const MString& string)
 
     PrimLookup lookup(SdfPath(strings2[0].asChar()), strings3[0].asChar(), obj);
 
+    static const MString uniqueKeyPrefix("uniquekey:");
+
     for(uint32_t j = 2; j < strings3.length(); ++j)
     {
+      if (strings3[j].substring(0, 10) == uniqueKeyPrefix)
+      {
+        auto keyStr(strings3[j].substring(10, strings3[j].length()));
```

Comment:
it may be a good idea to have 10 be defined as a variable with a descriptive name (maybe uniqueKeyPrefixLength) instead of being hardcoded twice here